custom-completions/zellij/zellij-completions.nu

```diff
@@ -81,6 +81,13 @@ def "nu-complete sessions" [] {
   ^zellij ls -n | lines | parse "{value} {description}"
 }
 
+def "nu-complete zellij layouts" [] {
+	ls ~/.config/zellij/layouts/ 
```

Comment:
I'm just wondering if this path is always correct across all platforms, like Windows for instance.

Reply:
Would something like this be better?

```nu
[($nu.default-config-dir | path dirname) "zellij" "layouts"] | path join
```

Reply:
It could be if zellij config is stored in that path on windows, linux, and macos. I'm not a zellij user so I'm not sure where exactly the config is stored. I peaked at the zellij code and noticed that it did run across those platforms, so it could be different but I'm not sure.

Reply:
ok. Maybe their references to Windows Terminal are just for WSL. Either way, the path where the config file is should probably be updated based on their docs https://zellij.dev/documentation/configuration#where-does-zellij-look-for-the-config-file

Reply:
oh, i see you've already done that. thanks!